packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/quanticStandaloneSearchBox.js

```diff
@@ -0,0 +1,298 @@
+// @ts-nocheck
+import {LightningElement, api, track} from 'lwc';
+// @ts-ignore
+import {
+  registerComponentForInit,
+  initializeWithHeadless,
+} from 'c/quanticHeadlessLoader';
+import {
+  NavigationMixin,
+  // @ts-ignore
+} from 'lightning/navigation';
+import {
+  STANDALONE_SEARCH_BOX_STORAGE_KEY
+} from 'c/quanticUtils';
+
+import search from '@salesforce/label/c.quantic_Search';
+
+const KEYS = {
+  ENTER: 'Enter',
+  ARROWUP: 'ArrowUp',
+  ARROWDOWN: 'ArrowDown',
+};
+const CLASS_WITH_SUBMIT =
+  'slds-combobox__form-element slds-input-has-icon slds-input-has-icon_right slds-input-has-fixed-addon';
+const CLASS_WITHOUT_SUBMIT =
+  'slds-combobox__form-element slds-input-has-icon slds-input-has-icon_left-right';
+
+export default class QuanticStandaloneSearchBox extends NavigationMixin (
+  LightningElement
+) {
+  
+  labels = {
+    search,
+  };
+
+  // Public props
```

Comment:
I like this. Maybe a JSDoc compatible notation would be even more helpful for us when we look into implementing it.

Reply:
Oh yea for sure!